WordPress/build.gradle

```diff
@@ -92,9 +92,9 @@ android {
             dimension "buildType"
             // Only set the release version if one isn't provided
             if (!project.hasProperty("versionName")) {
-                versionName "15.9.1"
+                versionName "15.9.2"
             }
-            versionCode 938
+            versionCode 943
```

Comment:
The latest `versionCode` on the Play Console was 942.

The issue that made CI fail [here](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/17778/workflows/dc3da90b-844d-4784-9234-33320a4e5918/jobs/79995) was that `versionCode` was the SemVer style value `15.9.2` instead of an integer.

I'm not sure how this happened in this instance, but I did the same thing once because I didn't read the prompt from the hotfix lane properly and input the `versionName` value when asked for `versionCode`.

Sounds like something we might want to guard against in the release toolkit. I had a quick look at the time and saw that Fastlane allows you to specify whether an input parameter should be an integer. But, when I tried to do it, the validation failed from CLI input. I haven't had the time to look deeper into it, yet.

cc @wordpress-mobile/release-managers.